File: 1511-count-number-of-teams/count-number-of-teams.cs
public class Solution {
    public int NumTeams(int[] rating) {
        int n = rating.Length, res = 0;
        for(int j = 0; j<n; j++){
            int ls=0,ll=0,rs=0,rl=0;
            for(int i=0; i<j; i++){
                if(rating[i]>rating[j]) ll++;
                else    ls++;
            }
            for(int k=j+1; k<n; k++){
                if(rating[k]>rating[j]) rl++;
                else    rs++;
            }
            res+=(ls*rl)+(ll*rs);
        }
        return res;
    }
}

File: 1511-count-number-of-teams/count-number-of-teams.py
class Solution:
    def numTeams(self, rating: List[int]) -> int:
        n=len(rating);ans=0
        for j in range(n):
            ls=ll=rs=rl=0
            for i in range(j):
                if rating[i]<rating[j]: ls+=1
                else:   ll+=1
            for k in range(j+1,n):
                if rating[k]>rating[j]: rl+=1
                else:   rs+=1
            ans+=ls*rl
            ans+=ll*rs
        return ans